nvfuser/__init__.py

```diff
@@ -360,15 +360,22 @@ def validate(
             if torch.is_floating_point(fusion_output) or torch.is_complex(
                 fusion_output
             ):
-                assert torch.allclose(
+                try:
+                    assert torch.allclose(
                     fusion_output, reference_output, atol=atol, rtol=rtol
-                ), f"Max error: {torch.abs(torch.max(fusion_output - reference_output))}, \
+                ), f"Max error nvFuser output[{inx}]: {torch.abs(torch.max(fusion_output - reference_output))}, \
+                    Absolute tolerance: {atol}, Relative tolerance: {rtol}"
+                except:
```

Comment:
Isn't this try-except block equivalent to `assert torch.allclose(xxx, yyy, 1.1*atol, 1.1*rtol)` ?

Reply:
Yes, it's not the final code (just a quick draft for verification). It will actually be based on whether the function argument specifies testing using relaxed tolerances.
But I can remove it in this version